Resources/Prototypes/_Goobstation/Entities/Clothing/OuterClothing/armour.yml

```diff
@@ -44,3 +44,68 @@
     sprite: _Goobstation/Clothing/OuterClothing/Armor/blueshield_cowboy_jacket.rsi
   - type: Item
     size: Normal
+
+- type: entity
+  parent: ClothingBlueshieldArmourVest
+  id: ClothingBlueshieldArmoredCoat
+  name: blueshield's armored coat
+  description: A coat for cold winter seasons with sewn in plates.
+  components:
+  - type: Sprite
+    sprite: _Goobstation/Clothing/OuterClothing/Armor/blueshield_coat.rsi
+  - type: Clothing
+    sprite: _Goobstation/Clothing/OuterClothing/Armor/blueshield_coat.rsi
+  - type: Item
+    size: Normal
+
+- type: entity
+  parent: ClothingBlueshieldArmourVest
+  id: ClothingBlueshieldArmoredCoatOpen
+  name: blueshield's open armored coat
+  description: An open coat for cold winter seasons with sewn in plates.
+  components:
+  - type: Sprite
+    sprite: _Goobstation/Clothing/OuterClothing/Armor/blueshield_coat_open.rsi
+  - type: Clothing
+    sprite: _Goobstation/Clothing/OuterClothing/Armor/blueshield_coat_open.rsi
+  - type: Item
+    size: Normal
```

Comment:
Make the coat toggleable between closed and open, example:
https://github.com/Simple-Station/Einstein-Engines/blob/aaadc3b7d9bf93200e03a596ce5fef159206ab88/Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml#L157-L175

You'll have to put the images of the open coat RSI to the normal coat RSI, with the following filenames changed:
`icon.png` -> `icon-open.png`
`equipped-OUTERCLOTHING.png` -> `open-equipped-OUTERCLOTHING.png` 
`inhand-left.png` -> `open-inhand-left.png` 
`inhand-right.png` -> `open-inhand-right.png`